src/main/java/org/gradle/profiler/buildops/BuildOperationDuration.java

```diff
@@ -0,0 +1,25 @@
+package org.gradle.profiler.buildops;
+
+import java.time.Duration;
+
+/**
+ * Carries total duration and count of one or more invocations of a build operation.
+ */
+public class BuildOperationDuration {
```

Comment:
How about we call this `BuildOperationExecutionData` or something? "Duration" is only part of the picture.

Reply:
Done